### PR TITLE
Profile Page

### PR DESCRIPTION
**Pull request by ardeshirj**
This PR ticket #28.

It includes the profile page from front-end team, and it populate the profile information from the database.

There are some minor issues where in some guides the reviews are not rendered correctly.  We can have a demo and go over it.

### CODE REVIEW

File: app/views/guides/show.html.erb [hunk 1]
```diff
@@ -1,94 +1,203 @@
...
+            <div class="col-xs-12 col-sm-5">
+              <img class="img img-circle img-responsive full-width"
+                src="img/guide-n-location/guide1.jpg"
+                style="position:relative; overflow: hidden;" alt="">
+            </div>
+            <div class="col-sm-offset-1 col-xs-12 col-sm-6">
+              <h1><strong><%= @guide.user.name %></strong></h1>
+              <h3><strong>Location: </strong><%= @guide.location %></h3>
+              <h3><strong>Sports: </strong><%= @guide.specialty %></h3>
+              <h3><strong>Pricing: </strong>$<%= @guide.rate %>/hour</h3>
```
Comment (beazley2) [inline app/views/guides/show.html.erb]: <h3><strong>Pricing: </strong><%= number_to_currency(guides.rate) %>/hour</h3>
will format the rate to 2 decimal pts and add the $ automatically.

File: app/views/guides/show.html.erb [hunk 2]
```diff
@@ -1,94 +1,203 @@
...
+              <h3><strong>Rating: </strong>
+                <% stars = overall_rating_stars(@guide.reviews) %>
```
Comment (beazley2) [inline app/views/guides/show.html.erb]: Not a big deal, but Active Record has a function that does this already:
'<% stars = @guide.reviews.average(:rating) %>`
Also, you removed half stars. Again, no big deal, but the code is already written for it.

**Comment by beazley2** [comment]
Other that that, looks great. Nice work.

File: app/controllers/guides_controller.rb [hunk 3]
```diff
@@ -30,7 +30,13 @@ def guide_params
                                   :location,
                                   :specialty,
                                   :rate,
-                                  :availability )
+                                  :sun_avail,
+                                  :mon_avail,
+                                  :tues_avail,
+                                  :wed_avail,
+                                  :thurs_avil,
```
Comment (beazley2) [inline app/controllers/guides_controller.rb]: Oh yeah, here was the minor typo I mentioned in our meeting yesterday. Should be :thurs_avail not :thurs_avil

**Comment by ardeshirj** [comment]
@beazley2 great review. Thank you.

**Comment by lewisric** [comment]
With all the changes I feel confident this will be fine

**Pull request merged**